api/test/service/store.test.js

```diff
@@ -72,9 +72,13 @@ test('store add returns signed url for uploading', async (t) => {
 
   const item = await getItemFromStoreTable(t.context.dynamoClient, alice, link)
   t.truthy(item)
-  t.truthy(item.uploadedAt)
-  t.truthy(item.proof)
-  t.truthy(item.uploaderDID)
+  t.is(typeof item?.uploadedAt, 'string')
+  t.is(typeof item?.proof, 'string')
+  t.is(typeof item?.uploaderDID, 'string')
+  t.is(item?.uploaderDID, account)
```

Comment:
account is now a bit of a trigger word for me. I see `account` and i mentally swap it with `space` and that trips me up here. Can we do
```suggestion
  t.is(item?.uploaderDID, alice.did())
```
and drop the word `account` from the tests pls?